### ISSUE TITLE
DataTable Single Select column throws "event not defined"  error. Error is live on PrimeNG Demo Page

### ISSUE THREAD

**Issue by rfboykin** [primefaces/primeng#1307]
Hey all,

I noticed some strange behavior with the single select datatable radio button column- when I try selecting a row, an `"EXCEPTION: event is not defined"` error is thrown. 

You can reproduce this yourself by going to the [PrimeNG Datatable Selection Page](http://www.primefaces.org/primeng/#/datatable/selection), opening your browser console and observing what happens when selecting a row from the "Radio Button" table. 

I've also tested these same settings with the multi-select checkbox, and for whatever reason those work. I also notice this error doesn't exist on single select rows (that is, when defining `[selectionMode]="single" `in the p-datatable tag and allowing for an entire row to be clickable in order to select it). To me these symptoms suggest the issue is in the p-radio button, but looking through the code on github I don't see what exactly the issue is or where the 'event' is not defined. 

Is anyone else getting this error? I couldn't find anything on Stack Overflow. 

Thank you for any help you can provide. 

Thanks,
Rob

**Comment by Mrtcndkn** [comment]
Hi,

I couldn't replicate your issue in locally.I opened datatable page i attached an event to the radio selection datatable then i select a row and event got fired.What should i do for replicating?

**Comment by rfboykin** [comment]
Hey,

Thanks for your reply, I'm seeing the error when using the page on Firefox 12. 

I just tested it in IE11 and do not see the error. 

On [PrimeNG's website](http://www.primefaces.org/primeng/#/datatable/selection) the error is thrown after selecting a radio button but it doesn't prevent the user from selecting another radio button. In my local environment however, selecting a radio button throws the error and prevents any other button from being selected - effectively breaking the page.  

Here is the code I'm using on my local environment: 
```
<p-dataTable #dt2 [value]="TActDocStatuses" [(selection)]="selectedTActDocStatus" (onRowSelect)="rowSelectedLog($event)">
  <p-column selectionMode="single" [style]="{'width':'42px'}"></p-column>
  <p-column field="nomenclature" header="Nomenclature" [sortable]="true" [editable]="true"></p-column>
</p-dataTable>
```

Where `rowSelectedLog($event)` resolves to: 
```
  rowSelectedLog(event?: any){
    console.log(event);
  }
```

Also, the full error and stacktrace I'm getting locally is: 

`EXCEPTION: Error in ./DataTable class DataTable - inline template:86:89 caused by: event is not defined`
```
[168]/g</DataTable.prototype.selectRowWithRadio@http://www.primefaces.org/primeng/dist/18.js:3:27487
anonymous/_View_DataTable37.prototype._handle_onClick_0_0@/DataTableModule/DataTable/component.ngfactory.js:2408:17
ur</EventEmitter.prototype.subscribe/i<@http://www.primefaces.org/primeng/dist/vendor.js:3:18950
c</SafeSubscriber.prototype.__tryOrUnsub@http://www.primefaces.org/primeng/dist/vendor.js:5:12881
c</SafeSubscriber.prototype.next@http://www.primefaces.org/primeng/dist/vendor.js:5:12185
u</Subscriber.prototype._next@http://www.primefaces.org/primeng/dist/vendor.js:5:11426
u</Subscriber.prototype.next@http://www.primefaces.org/primeng/dist/vendor.js:5:11073
p</Subject.prototype.next@http://www.primefaces.org/primeng/dist/vendor.js:5:6927
ur</EventEmitter.prototype.emit@http://www.primefaces.org/primeng/dist/vendor.js:3:18388
[168]/h</DTRadioButton.prototype.handleClick@http://www.primefaces.org/primeng/dist/18.js:3:16243
anonymous/_View_DTRadioButton0.prototype._handle_click_8_0@/DataTableModule/DTRadioButton/component.ngfactory.js:96:17
decoratePreventDefault/<@http://www.primefaces.org/primeng/dist/vendor.js:8:25471
se</DomEventsPlugin.prototype.addEventListener/i/<@http://www.primefaces.org/primeng/dist/vendor.js:9:25928
e</e</ZoneDelegate.prototype.invoke@http://www.primefaces.org/primeng/dist/polyfills.js:3:20869
cr</NgZone.prototype.forkInnerZoneWithAngularBehavior/this.inner<.onInvoke@http://www.primefaces.org/primeng/dist/vendor.js:3:21935
e</e</ZoneDelegate.prototype.invoke@http://www.primefaces.org/primeng/dist/polyfills.js:3:20807
e</n</Zone.prototype.runGuarded@http://www.primefaces.org/primeng/dist/polyfills.js:3:18353
cr</NgZone.prototype.runGuarded@http://www.primefaces.org/primeng/dist/vendor.js:3:20289
se</DomEventsPlugin.prototype.addEventListener/i@http://www.primefaces.org/primeng/dist/vendor.js:9:25897
e</e</ZoneDelegate.prototype.invokeTask@http://www.primefaces.org/primeng/dist/polyfills.js:3:21538
e</n</Zone.prototype.runTask@http://www.primefaces.org/primeng/dist/polyfills.js:3:18770
ZoneTask/this.invoke@http://www.primefaces.org/primeng/dist/polyfills.js:3:22711
```

Thanks for your help,
Rob

**Comment by Mrtcndkn** [comment]
I will review this situation and try to replicate it.I will inform you from here.

**Comment by dzhavat** [comment]
I just fixed that in my PR #1330
